README.md

```diff
@@ -9,37 +9,41 @@ Since we are using the lite version of raspbian, some extra packages need to be
 1. Install updates 
 
 ```
-sudo apt-get update 
-sudo apt-get upgrade
+sudo apt update 
+sudo apt upgrade
 ```
 2. Install Required Packages.
 
 Installation for python3-pip, raspotify, python3-tk, openbox
```

Comment:
@rsappia @utkut is there a special reason, why those commands are split and not in one single install command like: 

```sh
sudo apt install python-setuptools python3-setuptools python3-pip python3-tk redis-server openbox xorg lightdm x11-xserver-utils git
```

Reply:
No specific reason, i commited this thinking it would be a first draft, or to add new needed packages easier. Improvements are always welcomed.

Reply:
Ok thank you!